src/python/botan.py

```diff
@@ -837,7 +847,7 @@ def test_certs():
         print("Serial number:     %s" % hex_encode(cert.serial_number()))
         print("Authority Key ID:  %s" % hex_encode(cert.authority_key_id()))
         print("Subject   Key ID:  %s" % hex_encode(cert.subject_key_id()))
-        print("Public key bits:\n%s\n" % hex_encode(cert.subject_public_key_bits()))
+        print("Public key bits:\n%s\n" % b2a_base64(cert.subject_public_key_bits()))
```

Comment:
This doesn't seem right - the Python 3 docs say on b2a_base64 'The length of data should be at most 57 to adhere to the base64 standard.' which will not be true. I think we'd need the base64 module to do what you're looking for here which I do not want to deal with (TBH I'd rather just call botan's through ctypes since we can control that API).

TBH I don't think pretty printing here is very important, in the long run all of this code dies and is replaced by unittest code anyway. But I'll take the rest of the PR without this change.